test/performance/performance-tests.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script runs the end-to-end tests against Knative Serving built from source.
+# It is started by prow for each PR. For convenience, it can also be executed manually.
+
+# If you already have a Kubernetes cluster setup and kubectl pointing
+# to it, call this script with the --run-tests arguments and it will use
+# the cluster and run the tests.
+
+# Calling this script without arguments will create a new cluster in
+# project $PROJECT_ID, start knative in it, run the tests and delete the
+# cluster.
+
+source $(dirname $0)/../e2e-common.sh
+
+# Skip installing istio as an add-on.
+# Temporarily increasing the cluster size for serving tests to rule out
+# resource/eviction as causes of flakiness.
+initialize --skip-istio-addon --min-nodes=4 --max-nodes=4 --perf --cluster-version=1.21 "$@"
+
+header "Running tests"
+
+function run_kperf() {
+  run_go_tool knative.dev/kperf/cmd/kperf kperf "$@"
+}
+
+mkdir -p "${ARTIFACTS}/kperf"
+
+header "Running performance tests"
+export TIMEOUT=30m
+
+#  kperf options
+#  -b, --batch int                 Number of Knative Service each time to be created
+#  -c, --concurrency int           Number of multiple Knative Services to make at a time (default 10)
+#  -i, --interval int              Interval for each batch generation
+#  -n, --number int                Total number of Knative Service to be created
```

Comment:
Sorry I meant - can we drop `-b` in favour of `--batch` which is more verbose

Then we can drop this comment

Reply:
changed